fastestimator/network.py

```diff
@@ -143,39 +164,24 @@ def __init__(self, ops: Iterable[Union[TensorOp, Scheduler[TensorOp]]]):
         self.device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
         self.epoch_models = set()
 
-    def load_epoch(self, mode: str, epoch: int):
-        self.epoch_ops = get_current_ops(self.ops, mode, epoch)
+    def load_epoch(self, mode: str, epoch: int) -> List[TensorOp]:
+        epoch_ops = get_current_ops(self.ops, mode, epoch)
```

Comment:
should self.epoch_ops be removed from the init method?

Reply:
👍